precompiles/avsTask/methods.go

```diff
@@ -31,14 +31,7 @@ func (p Precompile) RegisterAVSTask(
 ) ([]byte, error) {
 	// check the invalidation of caller contract
 	callerAddress, _ := bech32.EncodeFromBase256("exo", contract.CallerAddress.Bytes())
-	avs, err := p.avsKeeper.GetAVSInfo(ctx, callerAddress)
-	if err != nil {
-		return nil, err
-	}
-
-	if avs.GetInfo() == nil {
-		return nil, fmt.Errorf(ErrNotYetRegistered, contract.CallerAddress)
-	}
+	_, err := p.avsKeeper.GetAVSInfo(ctx, callerAddress)
```

Comment:
Why disable the error checking? If the operator address is not exist, won't it return error? If for test purpose, we need to register the operator firstly before this task validate, right?

Reply:
fixed